spec/mailers/referee_mailer_spec.rb

```diff
@@ -12,11 +12,60 @@
     before { mail.deliver_now }
 
     it 'sends an email with the correct subject' do
-      expect(mail.subject).to include(t('reference_request.email.subject', candidate_name: candidate_name))
+      expect(mail.subject).to include(t('reference_request.subject.initial', candidate_name: candidate_name))
     end
 
     it 'sends an email with the correct heading' do
-      expect(mail.body.encoded).to include("give a reference for #{candidate_name}")
+      expect(mail.body.encoded).to include("Give a reference for #{candidate_name}")
+    end
+
+    it 'sends an email with the correct courses listed' do
```

Comment:
@tvararu @tijmenb We’ve added a list of selected courses to the reference request email, but @fofr was suggesting we test that this list appears and is correct. (We currently don’t test this in the application submitted email, either). I’ve replicated the code in the test, which is not a great test. How do you suggest I test this?

Reply:
Suggested offline: we can specify the choices explicitly in the setup for this test and then assert explicitly, without an `.each` loop.

Reply:
Updated with suggestion. Thanks @tvararu!